tasks/copy_to_archive_adapter/integration_test_copy_to_archive_adapter.py

```diff
@@ -0,0 +1,93 @@
+import os
+import time
+import unittest
+import uuid
+from unittest.mock import Mock
+
+import boto3
+
+import copy_to_archive_adapter
+
+
+class TestCopyToArchiveAdapter(unittest.TestCase):
+    def test_copy_to_archive_adapter_happy_path(self):
+
+        orca_recovery_bucket = os.environ["ORCA_RECOVERY_BUCKET"]
```

Comment:
We should add this to the script docs at a minimum, similar to `create_release.sh`.

Reply:
added in readme.md